src/mjolnir/ingest_transit.cc

```diff
@@ -257,18 +257,29 @@ void setup_stops(Transit& tile,
   auto onestop_id = isGenerated ? tile_stop.stop_id + "_" + to_string(node_type) : tile_stop.stop_id;
```

Comment:
to simplify our model of transit we require that every station (even if it doesnt in real life) has:
1. at least one in/egress
2. exactl one station
3. at least one platform

this makes handling connecting the data somewhat easiser and it models the highest level of complexity we want to support (even though bus stops dont need this in almost all circumstances). anyway... on the line above though you'll notice that what we do when the gtfs feed doesnt come with one of these things (either in/egress or platform, as it always has a station) we fake one so that our simplified model is fulfilled. when we fake one we have to give it a unique id so we  take its parent stations id and just add the type onto it (assuming real ones one use the type as a name).